### PR TITLE
Minor refactor to eliminate code redundancy

### PR DESCRIPTION
**Pull request by jamesgeorge007**
This is an attempt made to eliminate redundant code resulting in much concise code-base.

### CODE REVIEW

**Comment by jamesgeorge007** [comment]
cc @aslushnikov

File: lib/BrowserFetcher.js [hunk 1]
```diff
@@ -174,15 +172,16 @@ class BrowserFetcher {
...
+      chromePath = ['Chromium.app', 'Contents', 'MacOS', 'Chromium'];
     else if (this._platform === 'linux')
-      executablePath = path.join(folderPath, archiveName(this._platform, revision), 'chrome');
+      chromePath = 'chrome';
     else if (this._platform === 'win32' || this._platform === 'win64')
-      executablePath = path.join(folderPath, archiveName(this._platform, revision), 'chrome.exe');
+      chromePath = 'chrome.exe';
     else
       throw new Error('Unsupported platform: ' + this._platform);
+    const executablePath = path.join(folderPath, archiveName(this._platform, revision), Array.isArray(chromePath) ? ...chromePath : chromePath);
```
Comment (jamesgeorge007) [inline lib/BrowserFetcher.js:184]: It seems that the build fails as it doesn't recognize the spread operator (...)

**Comment by aslushnikov** [comment]
Hey @jamesgeorge007,

Why do you want to touch this?

**Comment by jamesgeorge007** [comment]
The specific code snippet appeared redundant at least IMO :thinking:

**Comment by aslushnikov** [comment]
> The specific code snippet appeared redundant at least IMO 🤔

That's subjective, but I don't have strong feeling here :-)

More importantly, I'd rather not change things that work unless there's a good reason to, e.g. fixing something or making things unquestionably better.

**Pull request closed without merging**